lib/dom.js

```diff
@@ -4,6 +4,117 @@
  * This source code is licensed under the MIT license found in the
  * LICENSE file in the root directory of this source tree.
  */
+/* Crypto */
+
+// This is a helper type to simplify the specification, it isn't an interface
+// and there are no objects implementing it.
+// https://developer.mozilla.org/en-US/docs/Web/API/BufferSource
+type BufferSource = ArrayBuffer | $ArrayBufferView;
+
+type DOMString = string;
+
+declare interface Crypto extends RandomSource {
+  subtle: SubtleCrypto; // readonly
```

Comment:
My head is currently in TypeScript land, but I am almost certain [this is the way to make read-only props on an interface](https://flow.org/en/docs/types/interfaces/#toc-interface-property-variance-read-only-and-write-only).

Reply:
That's indeed the way to make read-only props on an interface, but I did it this way, because in other parts of the file it was also done like this and I wasn't sure what to do with them so I stuck to the current way